.github/workflows/windows.yml

```diff
@@ -0,0 +1,137 @@
+          vcpkg integrate install
+      - name: "Build"
+        shell: pwsh
+        env:
+          PATH: '${env:PATH};C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64'
+        run: |
+          # build steps
+          cd visualc_net
+          MSBuild -m dosbox.sln -p:Configuration=Debug
+
+  build_msvc_release:
+    name: "MSVC 14 Release (win-2019)"
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v1
+      - name: "Log environment"
+        shell: pwsh
+        run: .\scripts\log-env.ps1
+      - name: "Install packages"
+        shell: pwsh
+        run: |
+          vcpkg install libpng sdl1 sdl1-net
+          vcpkg integrate install
+      - name: "Build"
+        shell: pwsh
+        env:
+          PATH: '${env:PATH};C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64'
+        run: |
+          # build steps
+          cd visualc_net
+          MSBuild -m dosbox.sln -p:Configuration=Release
+
+  build_msys2_gcc_64:
+    name: MSYS2 GCC-9 x86_64
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install msys2 environment
+        run: choco install msys2 --no-progress
+
+      - name: "Log environment"
+        run: C:\tools\msys64\usr\bin\bash -lc "cd d:/a/dosbox-staging/dosbox-staging; ./scripts/log-env.sh"
+
+      - name: Install dependencies
+        run: |
+          C:\tools\msys64\usr\bin\bash -lc "cd d:/a/dosbox-staging/dosbox-staging; ./scripts/list-build-dependencies.sh --output-file packages"
+          C:\tools\msys64\usr\bin\bash -lc "cd d:/a/dosbox-staging/dosbox-staging; xargs --arg-file=packages pacman -S --noconfirm"
+
+      - name: Build
+        run: C:\tools\msys64\usr\bin\bash -lc "cd d:/a/dosbox-staging/dosbox-staging; ./scripts/build.sh --fdo --bin-path /mingw64/bin"
+
+      - name: Summarize warnings
+        run: C:\tools\msys64\usr\bin\python3 ./scripts/count-warnings.py build.log
+
+  build_msys2_gcc_32:
+    name: MSYS2 GCC-9 i686
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install msys2 environment
+        run: choco install msys2 --no-progress
+
+      - name: "Log environment"
+        run: C:\tools\msys64\usr\bin\bash -lc "cd d:/a/dosbox-staging/dosbox-staging; ./scripts/log-env.sh"
+
+      - name: Install dependencies
+        run: |
+          C:\tools\msys64\usr\bin\bash -lc "cd d:/a/dosbox-staging/dosbox-staging; ./scripts/list-build-dependencies.sh --bit-depth 32 --output-file packages"
+          C:\tools\msys64\usr\bin\bash -lc "cd d:/a/dosbox-staging/dosbox-staging; xargs --arg-file=packages pacman -S --noconfirm"
+
+      - name: Build
+        run: C:\tools\msys64\usr\bin\bash -lc "cd d:/a/dosbox-staging/dosbox-staging; ./scripts/build.sh --bit-depth 32 --fdo --bin-path /mingw32/bin"
+
+      - name: Summarize warnings
+        run: C:\tools\msys64\usr\bin\python3 ./scripts/count-warnings.py build.log
```

Comment:
There is no need to specify python3 inside msys2 in here - python3 included with the VM will work just as well: `run: python3 scripts\count-warnings.py build.log`

Reply:
Just straight 'python3' didn't work with powershell. 
Adding `shell: bash` was no help either:
```
Run python3 ./scripts/count-warnings.py build.log
  python3 ./scripts/count-warnings.py build.log
  shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
  env:
    CHERE_INVOKING: yes
d:\a\_temp\798b2c42-25f6-4255-b7a8-a55bb9e180f0.sh: line 1: python3: command not found
```
Falling back to the full path for now.